### ISSUE TITLE
Feature Request: replacement for compass inline-image function base64 data url

### ISSUE THREAD

**Issue by VIEWSION** [bdkjones/CodeKit#585]
Hey,
since Compass is dead, I really miss the inline-image function to base64 files/images directly within Sass. It would be great to have this feature back on LibSass.

I found a similar custom Sass function here:
https://performancejs.com/post/ccffb43/Automatically-Generating-Base64-Inline-Images-With-SASS

would be great to have this built in natively!

**Comment by VIEWSION** [comment]
Any updates here?

**Comment by bdkjones** [comment]
Yep. I can add this capability, but it currently would apply only when compiling using Libsass. Since Dart Sass is the canonical Sass compiler, it would be best to have it available there as well. That MAY be possible, but it's going to take a lot of work.

**Comment by VIEWSION** [comment]
Thx... no worries - I was just wondering if something has changed here already. Do you have a workaround idea to get this done? At the moment I use PHP to replace this.... but I'd love to get this done with CodeKit again. Is there a way to add custom functions to the Sass compiler?
